proxy/proxy_test.go

```diff
@@ -295,6 +295,36 @@ func Test_buildProxyRequest_XForwardedHostHeader_WhenAlreadyPresent(t *testing.T
 	}
 }
 
+func Test_proxyRequest_ContentType_Header(t *testing.T) {
+	requestContentType := "x-www-form-urlencoded"
+	expectedContentTypeOfResponse := "application/json"
+	rr := httptest.NewRecorder()
+	req, err := http.NewRequest("POST", "/function/figlet", nil)
```

Comment:
Do something with that error - same as lower comment.

Reply:
Fixed